app/src/main/java/studyBuddy/Session.java

```diff
@@ -0,0 +1,133 @@
+package studyBuddy;
+
+import java.sql.Timestamp;
```

Comment:
this is good if we go the SQLite route for data storage but we may be able to just use java.util.Date